packages/terra-tabs/src/Tabs.jsx

```diff
@@ -161,7 +161,6 @@ class Tabs extends React.Component {
 
       clonedPanes.push(React.cloneElement(child, {
         className: cx([{ 'is-active': isActive }, child.props.className]),
-        'aria-selected': isActive,
```

Comment:
Does removing this aria role affect how the screen readers read this value?

Reply:
Added  aria-selected only for tab panes in 8dd92b0